### ISSUE TITLE
The application doesn't raise any exceptions, so the base error class is redundant

### ISSUE THREAD

**Issue by scepticalcat** [KarthickNarayana/toy_robot#3]
https://github.com/KarthickNarayana/toy_robot/blob/3d570a92d5c8b027e2456ec4f7bce47223719ecf/lib/toy_robot.rb#L10

I reaslise I've already mentioned bootstrapping the repo with Bundler, but this is a good example of an irrelevant artifact. The application doesn't raise any custom exceptions, so the defining of an namespaced `Error` class serves no purpose.

**Comment by KarthickNarayana** [comment]
Yes.  RubyMine created this stub file.  This is not an required line in the file.  

Exceptions can be thrown- while associating Robot to a Boundary - if no boundary is set or it is null - an illegalStateException is thrown.  The code in this sense is incomplete. I can always complete it as part of incorporating the feedback into the application
